### DISCUSSION TITLE
Not found Plots

### DISCUSSION THREAD

**Discussion by natamatov** [Chia-Network/chia-blockchain#6265] [Support]
i created plots over povershell
chia.exe plots create -k 32 ...
Then I added a directory with plots to the program, but the program gives me Not found Plots.
I used these plots on HPOOL.
Can I use them in solo mining now or not?

**Comment by BrandtH22** [comment]
@natamatov , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Note - the OG hpool pool required users to provide their keys to the pool, it is recommended to fully replot with new keys when migrating off from hpools OG pool.

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
